tests/DiscussionEmbed.js

```diff
@@ -67,15 +73,23 @@ test('Inserts the script correctly', () => {
     expect(scriptQuery[0].src).toEqual('https://testing.disqus.com/embed.js');
 });
 
-test('Inserts the script correctly when passed an SSO config', () => {
-    const TEST_CONFIG = DISQUS_CONFIG;
+test('Inserts the script correctly when passed a remoteAuthS3 string and API Key with an SSO config', () => {
+    const TEST_CONFIG = DISQUS_CONFIG_WITH_SSO_AUTH;
     TEST_CONFIG.sso = SSO_CONFIG;
```

Comment:
This is old, but setting `TEST_CONFIG.sso` this way will set `DISQUS_CONFIG.sso` as well and affect the other tests (including the one you added). Could you fix it since you're touching these tests already?